base_layer/core/tests/mempool.rs

```diff
@@ -751,10 +751,9 @@ fn service_request_timeout() {
 }
 
 #[test]
-#[ignore = "Flaky test that needs to be fixed"]
 #[allow(clippy::identity_op)]
 fn block_event_and_reorg_event_handling() {
-    // #flaky, this test seems to fail after submiting block B2A to bob.
+    // #flaky, this test seems to fail after submitting block B2A to bob.
```

Comment:
Sorry I forgot to ask to remove this comment as well!

Reply:
My bad, my spellchecker just noticed that mistake, next time I will read the comments more carefully.

Reply:
No worries 🙃